packages/two_dimensional_scrollables/lib/src/table_view/table.dart

```diff
@@ -667,12 +667,14 @@ class RenderTableViewport extends RenderTwoDimensionalViewport {
   }) {
     // TODO(Piinks): Assert here or somewhere else merged cells cannot span
     // pinned and unpinned cells (for merged cell follow-up), https://github.com/flutter/flutter/issues/131224
-    double yPaintOffset = -offset.dy;
+    late double yPaintOffset = -offset.dy;
     for (int row = start.row; row <= end.row; row += 1) {
       double xPaintOffset = -offset.dx;
       final double rowHeight = _rowMetrics[row]!.extent;
+      yPaintOffset += _rowMetrics[row]!.configuration.padding.leading;
```

Comment:
nit: maybe get the rowMetric only once to avoid extra null checks:

```suggestion
      final rowMetric = _rowMetrics[row]!;
      final double rowHeight = rowMetric.extent;
      yPaintOffset += rowMetric.configuration.padding.leading;
```

Reply:
Same for columnMetrics and the other rowMetric access below.